client_code/forms/routes/__init__.py

```diff
@@ -1,2 +1,2 @@
-from . import ballot, vote
-from .Home import Home
+from . import ballot, vote  # noqa F401
```

Comment:
what is this?

Reply:
It's a comment that stops flake8 compatible linters from complaining about an unused import. F401 is the error code for that.

https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

As it's just a comment, it has no effect on execution.

Reply:
I like to get linting errors to zero so I can tell when I've intoduced one!